snippets/use_active_record_false.rb

```diff
@@ -0,0 +1,39 @@
+require "bundler/inline"
+
+gemfile(true) do
+  source "https://rubygems.org"
+
+  git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+  # Those Gemfiles carefully pick the right versions depending on
+  # settings in the ENV, `.rails-version` and `maintenance-branch`.
+  eval_gemfile './Gemfile-sqlite-dependencies'
+  eval_gemfile './Gemfile-rspec-dependencies'
+  eval_gemfile './Gemfile-rails-dependencies'
+
+  gem "ammeter"
+  gem "rspec-rails", path: "./"
+end
```

Comment:
Do we want to eventually extract this to `snipets/gemfile.rb` or similar? And require that?